docs/source/index.rst

```diff
@@ -75,6 +77,12 @@ or install it using your system package manager, e.g.::
 
     sudo apt-get install python3-pyqt5
 
+or::
+
+    sudo apt-get install python-pyqt
+    sudo apt-get install python3-pyqt
+
+if PyQt4 is desired.
```

Comment:
Then we should remove the reference to Ubuntu and `python3-pyqt5` as well because (as I said) it's misleading: casual users will believe that `qtconsole` only works on Python 3 and PyQt5.

For what is worth, we don't provide specific installation instructions for any Linux distribution in Spyder, so I'm fine with going with that too here :-)